user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+example for several organizations would be a franchisor that has several franchisees - the best way to represent each 
+of them in OroCRM is a separate organization. Even a freelance entrepreneur working with several different businesses 
+from his computer will benefit using a separate organization to represent each of them in OroCRM.
+
+Subject to the work done in the company, some users may have access to several organizations 
+(work with multiple organization) or to all records across all organizations that are available to them 
+(work from a system organization). This is described below. 
+
+    
+.. _user-guide-getting-started-change-organization:
+
+Work with Multiple Organizations
+--------------------------------
+
+In many cases a user has access to a single organization. A sales representative in the U.S. office is likely to have no 
+access to the company’s French division.
+
+However, users can be assigned to multiple organizations just as they can be assigned to multiple business units and 
+will get access to the records in different organizations. For example, if you are working at the worldwide help desk 
+of a multinational organization, you should be able to access customer records in all the organization within your 
+company in order to open cases based on the customer requests. Similarly, if you are the CEO of a big enterprise 
+comprised of multiple independent divisions, you would like to have access to every one of them in order to check their 
+performance.
+
+As organizations are often much bigger and more separated than business units, the user will have to switch to the 
+necessary organization to work with data in each one of them.
+
+
+Switching Between Organizations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The name of an organization you are currently working with is displayed in the top left corner of the screen. If you 
+have access to multiple organizations, upon your first login you will see a note with your current organization. Three 
+dots to the left are the organization selector.
+
+|
+
+.. image:: ./img/multi_org/multi_org_select.png
+
+|
+
+Click the selector, to see a list of organizations you have access to. Your current organization will be displayed in 
+bold. If a System organization exists, it will be shown at the top of the list and shifted left
+
+|
+
+.. image:: ./img/multi_org/multi_org_choice.png
+
+|
+
+To switch to another organization, click its name in the list.
+
+
+Work with a System Organization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A System Organization provides users with simultaneous access to all the records of all the organizations they are 
+assigned to. There cannot be more than one system organization in OroCRM. 
+
+When working in a system organization all grids will have an additional ‘Organization"* column. The *"Organization"* field 
+will be shown for each record viewed. The organization field is not editable.
```

Comment:
Edit: When working in a system organization, all grids will have an additional *"Organization"* column. The *"Organization"* field will be shown for each record viewed, and is not editable.